### PR TITLE
GPII-2805 - Add backup testing procedure to stg environment

### PR DESCRIPTION
**Pull request by amatas**
Part of [GPII-2805](https://issues.gpii.net/browse/GPII-2805)

This PR adds the description of a periodically procedure applied to the `stg` environment of the infrastructure.

[Rendered version](https://github.com/gpii-ops/gpii-infra/blob/53d5aa9348fd01cea33e66f3203196631ea53be4/gcp/README.md#stg)

### CODE REVIEW

File: gcp/README.md [hunk 1]
```diff
@@ -138,6 +138,8 @@ Deploying to `stg` verifies that the gpii-infra code that worked to create a `de
 
 Because `stg` emulates production, it will (in the future) allow us to run realistic end-to-end tests before deploying to `prd`.
 
+The `stg` environment is almost the same environment as `prd`, this fact makes `stg` the proper environment to test some recovering procedures that the ops team must perform periodically. Like the restoration of the persistance layer. **The ops team will perform a restoration of the persistance layer every first monday of the month as part of the procedures verification needed for the FERPA compliance, so expect an outage of about 30 min at that time of the services in `stg`**.
```
Comment (mrtyler) [inline gcp/README.md]: "Like the restoration of the persistance layer." is not a sentence.

Comment (mrtyler) [inline gcp/README.md]: persistence

Comment (mrtyler) [inline gcp/README.md]: I would unbold -- bolding a long sentence adds visual clutter; it's a minor point, not worth drawing a reader's attention.

Comment (mrtyler) [reply gcp/README.md]: I would move most of this under the last bullet point in the SLO section (this is what I requested yesterday).

File: gcp/README.md [hunk 2]
```diff
@@ -138,11 +138,14 @@ Deploying to `stg` verifies that the gpii-infra code that worked to create a `de
 
 Because `stg` emulates production, it will (in the future) allow us to run realistic end-to-end tests before deploying to `prd`.
 
+The `stg` environment is almost the same environment as `prd`, this fact makes `stg` the proper environment to test some recovering procedures that the ops team must perform periodically.
```
Comment (natarajaya) [inline gcp/README.md]: First part of this sentence duplicates the statement on line 139. Maybe combine them?

File: gcp/README.md [hunk 3]
```diff
@@ -138,11 +138,14 @@ Deploying to `stg` verifies that the gpii-infra code that worked to create a `de
...
 Because `stg` emulates production, it will (in the future) allow us to run realistic end-to-end tests before deploying to `prd`.
 
+The `stg` environment is almost the same environment as `prd`, this fact makes `stg` the proper environment to test some recovering procedures that the ops team must perform periodically. 
+
 #### Service Level Objective
 
 * No guarantee of service for development environments (stg is a development environment).
 * Ops team receives alerts about problems, and addresses them when convenient for the team (i.e. we won't wake up an on-call engineer, but we will investigate during business hours)
 * Ops team makes an effort to keep stg stable and available for ad-hoc testing, but may disrupt the environment at any time for our own testing.
+* Ops team will perform a restoration of the persistence layer every first monday of the month as part of the procedures verification needed for the FERPA compliance, so expect an outage of about 30 min at that time of the services in `stg`.
```
Comment (natarajaya) [inline gcp/README.md]: At what time? Also, I feel like outage duration may vary (we only had 1 test performed so far). Maybe just state that the beginning and the end of the outage is going to be announced in #ops channel? I would replace "procedures verification" with "backup testing process".

**Pull request merged**